### PR TITLE
[Justin] - Sage - Create a Lottery dApp using Celo Composer and Redstone's Randomness

### PR DESCRIPTION
**Pull request by cjustinobi**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md [hunk 1]
```diff
@@ -0,0 +1,691 @@
+---
+title: Create a Lottery dApp using Celo Composer and Redstone's Randomness
+description: Solidity smart contracts can use randomness to ensure that lottery outcomes are fair and unbiased, hence the use of Redstone randomness, which allows for transparency in the implementation.
+authors:
+  - name: Justin Obi
+    title: Software Engineer / Technical Writer
+    url: https://github.com/cjustinobi
+    image_url: https://github.com/cjustinobi.png
+tags: [celosage, intermediate, react, hardhat, oracle]
```
Comment (Julius170) [inline blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md:9]: can you add all the tags you have in your user.tsx file here too, it has to be the same thing

File: blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+This file imports the `ethers` and `Contract` modules from the `ethers` package, the `WrapperBuilder` module from the `redstone-evm-connector` package. The `priceToWei` function from another file is also imported and used to convert a non-Wei price to Wei. The contractAddress variable is the address of the smart contract we got when we deployed our smart contract.
+
+## Conclusion
+
+Congratulations! You should be able to create a dApp that involves use of random numbers by utilizing Redstone oracle and Celo Composer speedily. [Here](https://github.com/cjustinobi/celo-redstone-lottery) is the link to the complete code.
+
+
+## About the Author​
+
+A software engineer, co-founder, Africinnovate, and a Web3 enthusiast. I used to call myself VueDetective. Connect with me on LinkedIn and Github
```
Comment (Julius170) [inline blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md:685]: Can you add links to your LinkedIn and your Github profile here

File: blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md [hunk 3]
```diff
@@ -0,0 +1,691 @@
...
+
+## References​
+[How to use Redstone Randomness](https://docs.redstone.finance/docs/smart-contract-devs/randomness)
```
Comment (Julius170) [inline blog/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness/2023-04-24-create-a-lottery-dapp-using-celo-composer-and-redstone-randomness.md:688]: Can you make this a bullet point

**Review by Julius170 — COMMENTED** [review]
Hi @cjustinobi Just left some comments, once you've addressed them you'll be all set to go

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**